tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,457 @@
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
+                }
+            ],
+            "programmingLanguage": {
+                "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl"
+            },
+            "version": "1.0.0",
+            "yevisId": "c13b6e27-a4ee-426f-8bdb-8cf5c4310bad"
+        },
+        {
+            "@id": "https://w3id.org/cwl/vv1.0/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://www.commonwl.org/",
+            "@type": [
+                "WebPage"
+            ]
+        },
+        {
+            "@id": "https://w3id.org/workflowhub/workflow-ro-crate#cwl",
+            "@type": "ComputerLanguage",
+            "alternateName": "CWL",
+            "identifier": {
+                "@id": "https://w3id.org/cwl/vv1.0/"
+            },
+            "name": "Common Workflow Language",
+            "url": {
+                "@id": "https://www.commonwl.org/"
+            },
+            "version": "v1.0"
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md",
+            "@type": [
+                "WebPage"
+            ],
+            "name": "README.md"
+        },
+        {
+            "@id": "exe/fastqc.cwl",
+            "@type": [
+                "File",
+                "FormalParameter",
+                "WorkflowAttachment",
+                "SoftwareSourceCode"
+            ],
+            "contentSize": 577,
+            "dateModified": "2023-01-13T09:13:29.886652",
+            "encodingFormat": "text/plain",
+            "lineCount": 33,
+            "sha512": "af6d23d05f8b42739feb4db4f3222e0201f7186953a4200bc3a72668589db672d738f27275eb5298cd2d4a1f5f3aa854f5f56a8561f5a8217a7f1c9663627b95",
+            "url": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/fastqc.cwl"
+        },
+        {
+            "@id": "exe/trimmomatic_pe.cwl",
+            "@type": [
+                "File",
+                "FormalParameter",
+                "WorkflowAttachment",
+                "SoftwareSourceCode"
+            ],
+            "contentSize": 1570,
+            "dateModified": "2023-01-13T09:13:30.521735",
+            "encodingFormat": "text/plain",
+            "lineCount": 66,
+            "sha512": "cc384f32389e588706860139d843194bef48d53081183f12fc3ed32d3bc56bfca868c2965d5e9dd4e50a66e913b7b3afc0e0f9766353a14ad87f3b4a33b2ed2e",
+            "url": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimmomatic_pe.cwl"
+        },
+        {
+            "@id": "#26cee039-7a37-4647-924f-ee2187304ef3",
+            "@type": "CreateAction",
+            "endTime": "2023-01-13T09:15:33",
+            "exitCode": 0,
```

Comment:
We're using `actionStatus` to indicate the outcome, see https://www.researchobject.org/workflow-run-crate/profiles/process_run_crate#requirements